src/canvas-extractor/tests/mapping/test_users.py

```diff
@@ -76,7 +76,7 @@ def test_then_email_address_is_mapped(result):
             assert result.at[0, "EmailAddress"] == "Kyle.Hughes@studentgps.org"
 
         def test_then_it_should_have_empty_SourceCreateDate(result):
-            assert result.at[0, "SourceCreateDate"] == "2020/09/14 11:54:18"
+            assert result.at[0, "SourceCreateDate"] == "2020-09-14 11:54:18"
```

Comment:
Side effect of standardizing the date conversion: found an inconsistency in our previous handling.

Reply:
Nice